ConferencesApp/ConferenceTableViewController.swift

```diff
@@ -10,11 +10,16 @@ import UIKit
 import MapKit
 import Haneke
 import AddressBook
+import EventKit
 
 class ConferenceTableViewController: UITableViewController, MKMapViewDelegate {
     var conference = Conference()
-
-
+    var conferenceDataStore : ConferenceDataStore?
+    var conferenceName: String!
+    var startDate: NSDate!
+    var endDate: NSDate!
+    var calendarEventID : String!
```

Comment:
Colon Violation: When specifying a type, always associate the colon with the identifier